Sources/Core/ModernAVPlayer.swift

```diff
@@ -110,6 +111,16 @@ public final class ModernAVPlayer: NSObject, MediaPlayer {
         }
         context.loadMedia(media: media, autostart: autostart)
     }
+   
+    /**
+     Replaces the current player metadata with the new metadata
```

Comment:
metedata belong to item, not player right nope ?
so Replaces the current item metadata with the new metadata ?

Reply:
yes